### PR TITLE
subject(param_range):add Readme

### PR DESCRIPTION
**Pull request by Hamzaelkhatri**

### CODE REVIEW

File: subjects/paramrange/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+## param-range
```
Comment (zainabdnaya) [inline subjects/paramrange/README.md:1]: # param-range 

just one Hashtag

File: subjects/paramrange/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+### instructions
+
+- Write a program that takes a number in the arguments and prints the max and min.
```
Comment (zainabdnaya) [inline subjects/paramrange/README.md:5]: I guess no need for '-'

File: subjects/paramrange/README.md (hunk 1 above)
Comment (Hamzaelkhatri) [reply subjects/paramrange/README.md:1]: no should be two hashtag

**Review by zainabdnaya — APPROVED** [review]
All good

**Review by davidrobert99 — APPROVED** [review]

**Pull request merged**